.github/CODEOWNERS

```diff
@@ -46,9 +46,13 @@
 /experimental/dds/tree/src                     @microsoft/fluid-cr-tree
 
 # Fluid Devtools source
-/packages/tools/devtools/**/src                   @microsoft/fluid-cr-devtools
+/packages/tools/devtools/**/src                @microsoft/fluid-cr-devtools
+
+# Public and Legacy API changes
+/**/api-report/*.public.api.md                 @microsoft/fluid-cr-api
+/**/api-report/*.alpha.api.md                  @microsoft/fluid-cr-api
 
 # Package version files are not generally interesting to review by teams signing up for source tree ownership.
 # If a release team wanted to sign up to block on changes relevant for package release,
 # that would be the appropriate owner here. For now, explicitly give these files no owner.
-/**/packageVersion.ts
\ No newline at end of file
+/**/packageVersion.ts
```

Comment:
Benign: vs code automatically added a trailing LF on save.  Not going to fight it.